workgroup/aws-athena-workgroup.json

```diff
@@ -27,7 +27,8 @@
             "required": [
```

Comment:
Should we add ListTagResource to read permission in the handler section?

Reply:
Good catch. I'm not sure how this was working without it but it should definitely be there, I'll add it in